libraries/AP_HAL_ChibiOS/Util.cpp

```diff
@@ -599,3 +600,50 @@ void Util::uart_info(ExpandingString &str)
 #endif
 #endif // HAL_NO_UARTDRIVER
 }
+
+/*
+  log info on stack usage. Called at 1Hz by logging thread, logs next
+  thread on each call
+*/
+void Util::log_stack_info(void)
+{
```

Comment:
This looks like a copy of the @SYS/threads.txt code - any way the two can be combined to save flash?